tests/ft_calloc_test.cpp

```diff
@@ -20,7 +21,31 @@ int main(void)
 	char e[] = {0, 0, 0, 0};
 	/* 1 */ check(!memcmp(p, e, 4));
 	/* 2 */ mcheck(p, 4); free(p); showLeaks();
-	/* 3 */ check(ft_calloc(SIZE_MAX, SIZE_MAX) == NULL);
+	/* 3 */ check(ft_calloc(SIZE_MAX, SIZE_MAX) == NULL); showLeaks()
+
+	/* @evportel */
+	/* The following tests are not supported by the function's documentation. 
+ 	* But some effects returned in the trait by Moulinette so the following 
+  	* tests were implemented. */
+	/* 4 */ check(ft_calloc(INT_MAX, INT_MAX) == NULL);
+	p = ft_calloc(INT_MIN, INT_MIN);
+	/* 5 */ check(p == NULL); free(p); //showLeaks();
+	p = ft_calloc(0, 0);
+	/* 6 */ check(p != NULL); free(p); //showLeaks();
+	p = ft_calloc(0, 5);
+	/* 7 */ check(p != NULL); free(p); //showLeaks();
+	p = ft_calloc(5, 0);
+	/* 8 */ check(p != NULL); free(p); //showLeaks();
+	/* 9 */ check(ft_calloc(-5, -5) == NULL); showLeaks();
```

Comment:
Could you please enabled showLeaks for all your tests and directly call ft_calloc like test 9 for every case the function should return null please (5, 12, 13).

Reply:
@evertonportela

Reply:
Updated @Tripouille, sorry for the delay, ✌️